### PR TITLE
add doc strings to udSDK.py methods

### PR DESCRIPTION
**Pull request by bwockner**

### CODE REVIEW

File: src/udSDK/udSDK.py [hunk 1]
```diff
@@ -12,8 +12,11 @@
 
 
 class UdException(Exception):
+  """
+  Exception class wrapping udError return values. udError value can be accessed as args[1]
```
Comment (samsinsane) [inline src/udSDK/udSDK.py:16]: Should this say `code` instead of `args[1]`?

File: src/udSDK/udSDK.py [hunk 2]
```diff
@@ -255,6 +281,9 @@ class udStdAttribute(IntEnum):
 
 
 class udStdAttributeContent(IntEnum):
+  """
+  Standar attributes as a bit field. The presence of each attribute is indicated by its bit being set
```
Comment (samsinsane) [inline src/udSDK/udSDK.py:285]:
```suggestion
  Standard attributes as a bit field. The presence of each attribute is indicated by its bit being set
```

File: src/udSDK/udSDK.py [hunk 3]
```diff
@@ -351,6 +392,9 @@ def __repr__(self):
 
 
 class udAttributeSet(ctypes.Structure):
+  """
+  Set of udAttributes. Used when describing the contents of a udPointCLoud or a udPointBuffer
```
Comment (samsinsane) [inline src/udSDK/udSDK.py:396]:
```suggestion
  Set of udAttributes. Used when describing the contents of a udPointCloud or a udPointBuffer
```

File: src/udSDK/udSDK.py [hunk 4]
```diff
@@ -1310,6 +1412,10 @@ class _udPointBufferF64(ctypes.Structure):
   dtype = "f8"
 
   def __init__(self, maxPoints=0, attributeSet=None, pStruct=None):
+    """
+    Creates a point buffer with maxPoints allocated to it with the attributes defined by attributeSet.
+    If pStruct is defined instead instantiate the object with the internal value loacted at this address
```
Comment (samsinsane) [inline src/udSDK/udSDK.py:1417]:
```suggestion
    If pStruct is defined instead instantiate the object with the internal value located at this address
```

File: src/udSDK/udSDK.py [hunk 5]
```diff
@@ -1380,6 +1489,11 @@ def pointcloud(self, pointcloud: udPointCloud):
     _HandleReturnValue(self.udQueryContext_ChangePointCloud(self.pQueryContext, pointcloud.pPointCloud))
 
   def execute(self, points: udPointBufferF64):
+    """
+    Fill the point buffer with points matching the query from the pointcloud. Call repeatedly to more points matching
```
Comment (samsinsane) [inline src/udSDK/udSDK.py:1493]:
```suggestion
    Fill the point buffer with points matching the query from the pointcloud. Call repeatedly to get more points matching
```

I'm not certain if this is the correct change, but the sentence as-is doesn't make sense.

File: src/udSDK/udSDK.py [hunk 6]
```diff
@@ -1391,7 +1505,8 @@ def _destroy(self):
 
   def load_all_points(self, bufferSize=100000):
     """
-    This loads all points from the UDS into the resultsBuffers array
+    This loads all points matching the query into a list of point buffers of size buffersize.
```
Comment (samsinsane) [inline src/udSDK/udSDK.py:1508]:
```suggestion
    This loads all points matching the query into a list of point buffers of size bufferSize.
```

**Review by samsinsane — CHANGES_REQUESTED** [review]

**Comment by OsiJr** [comment]
Merging as all of Sams feedback was fixed.

**Pull request merged**